Kernel/Arch/DebugOutput.h

```diff
@@ -8,6 +9,7 @@
 
 namespace Kernel {
 
+void platform_debug_output(char ch);
 void debug_output(char ch);
```

Comment:
Where is the purpose in this distinction if one just calls the other in all cases? As far as I can tell, kprintf is the only user of this API anyways. You can have the declaration shared, and the definition in two different cpp files for two different architectures.

Reply:
you can see the difference in usage in `kprintf.cpp` which does not care for the platform it is run on.

as for the first of your points: if i understood correctly you claim that i always call `platform_debug_output` from `debug_output`? thatʼs only true for RPi (Aarch64) branch. implementation for x86-64 differs, cf. `DebugOutput` in `x86_64`.

that is still subject for a change as one would probably like to see implementations more aligned. yet i considered any more changes here as out of scope — mostly due to the fact my pull requests already tended to take considerable amount of time (mostly due to the fact they could be pretty big). 🫣

is there something i should amend or are you satisfied with my explanations and this is good as is? 😅

Reply:
I'm not fully convinced but I don't want to block the PR on something that's this trivial to change.

Reply:
Hmmm, I think I understand now. Consider this comment resolved :^)

Reply:
oh! 🎩